### PR TITLE
replaced deprecated terms in various files as part of inclusive naming

### PR DESCRIPTION
**Pull request by CatherineKiiru**
## Removal and replacement of offensive terms as part of the inclusive naming project
[x] Make sure you are opening from a topic/feature/bugfix branch (right side) and not your main branch!
[x] Ensure that the pull request title represents the desired changelog entry
[x] Please describe what you did
[x] Link to relevant issues in GitHub or Jira
[x] Link to relevant pull requests, esp. upstream and downstream changes
~~Ensure you have provided tests - that demonstrates feature works or fixes the issue~~

### CODE REVIEW

File: pipeline-model-definition/src/main/groovy/org/jenkinsci/plugins/pipeline/modeldefinition/model/Root.groovy [hunk 1]
```diff
@@ -26,7 +26,7 @@ package org.jenkinsci.plugins.pipeline.modeldefinition.model
 import edu.umd.cs.findbugs.annotations.SuppressFBWarnings
 import groovy.transform.EqualsAndHashCode
 import groovy.transform.ToString
-import org.jenkinsci.plugins.scriptsecurity.sandbox.whitelists.Whitelisted
+import org.jenkinsci.plugins.scriptsecurity.sandbox.s.Whitelisted
```
Comment (aHenryJard) [inline pipeline-model-definition/src/main/groovy/org/jenkinsci/plugins/pipeline/modeldefinition/model/Root.groovy:29]: This change is code change, and seems to be an error to me, you may want to revert this part.

File: pipeline-model-definition/src/test/java/org/jenkinsci/plugins/pipeline/modeldefinition/AbstractModelDefTest.java [hunk 2]
```diff
@@ -596,10 +596,10 @@ public WorkflowRun getRun() {
     }
 
     public class EnvBuilder {
-        private final Slave agent;
```
Comment (aHenryJard) [inline pipeline-model-definition/src/test/java/org/jenkinsci/plugins/pipeline/modeldefinition/AbstractModelDefTest.java:599]: This part is code change and should be reverted too.

File: pipeline-model-definition/src/test/java/org/jenkinsci/plugins/pipeline/modeldefinition/AbstractModelDefTest.java [hunk 3]
```diff
@@ -596,10 +596,10 @@ public WorkflowRun getRun() {
...
+        private final agent;
         private Map<String, String> env;
 
-        protected EnvBuilder(Slave agent) {
+        protected EnvBuilder(agent) {
```
Comment (aHenryJard) [inline pipeline-model-definition/src/test/java/org/jenkinsci/plugins/pipeline/modeldefinition/AbstractModelDefTest.java:602]: This part is code change and should be reverted too.

**Review by aHenryJard — CHANGES_REQUESTED** [review]

File: pipeline-model-definition/src/main/groovy/org/jenkinsci/plugins/pipeline/modeldefinition/model/Root.groovy (hunk 1 above)
Comment (MarkEWaite) [reply pipeline-model-definition/src/main/groovy/org/jenkinsci/plugins/pipeline/modeldefinition/model/Root.groovy:29]: Agreed that this needs to be reverted.  Compilation will fail with this change

File: pipeline-model-definition/src/test/java/org/jenkinsci/plugins/pipeline/modeldefinition/AbstractModelDefTest.java (hunk 2 above)
Comment (MarkEWaite) [reply pipeline-model-definition/src/test/java/org/jenkinsci/plugins/pipeline/modeldefinition/AbstractModelDefTest.java:599]: Agreed.  This needs to be reverted

File: pipeline-model-definition/src/test/java/org/jenkinsci/plugins/pipeline/modeldefinition/AbstractModelDefTest.java (hunk 3 above)
Comment (MarkEWaite) [reply pipeline-model-definition/src/test/java/org/jenkinsci/plugins/pipeline/modeldefinition/AbstractModelDefTest.java:602]: Agreed.

**Review by MarkEWaite — CHANGES_REQUESTED** [review]

**Comment by MarkEWaite** [comment]
This pull request can be closed.  #522 has been merged and includes the relevant changes from this pull request

**Comment by jglick** [comment]
closing as per https://github.com/jenkinsci/pipeline-model-definition-plugin/pull/521#issuecomment-1139115222

**Pull request closed without merging**